src/drone_controller/calculation_layer/thrust_calculator.py

```diff
@@ -0,0 +1,20 @@
+# pylint: disable=too-few-public-methods
+class ThrustCalculator:
```

Comment:
I think the ThrustCalculator should be Drone specific. eg. ThrustCalculatorQuatrocopter.
I believe calculating the correct thrust highly depends on the drone setup.